File: problemas/iniciante/1134/1134.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

let [alcool, gasolina, diesel] = [0, 0, 0];
while (lines.length) {
    let codigo = parseInt(lines.shift());

    if (codigo === 4) {
        break;
    }

    switch (codigo) {
        case 1: ++alcool;
            break;
        case 2: ++gasolina;
            break;
        case 3: ++diesel;
            break;
        default: break;
    }
}

console.log("MUITO OBRIGADO");
console.log(`Alcool: ${alcool}`);
console.log(`Gasolina: ${gasolina}`);
console.log(`Diesel: ${diesel}`);

File: problemas/iniciante/1134/1134.ml
let alcool = ref 0 in
let gasolina = ref 0 in
let diesel = ref 0 in
let () =
  try
    while true do
      Scanf.scanf "%d\n" (fun codigo ->
          if codigo = 1 then alcool := !alcool + 1
          else if codigo = 2 then gasolina := !gasolina + 1
          else if codigo = 3 then diesel := !diesel + 1
          else if codigo = 4 then raise End_of_file)
    done
  with End_of_file -> ()
in
Printf.printf "MUITO OBRIGADO\n";
Printf.printf "Alcool: %d\n" !alcool;
Printf.printf "Gasolina: %d\n" !gasolina;
Printf.printf "Diesel: %d\n" !diesel
